### PR TITLE
Fix DaprSecretStoreConfigurationProvider

### PR DESCRIPTION
**Pull request by Fabian-Schmidt**
# Description

Fix Copy and Paste issue in `DaprSecretStoreConfigurationProvider`.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #615

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: test/Dapr.Extensions.Configuration.Test/DaprSecretStoreConfigurationProviderTest.cs [hunk 1]
```diff
@@ -437,7 +437,7 @@ private async Task SendBulkResponseWithSecrets(Dictionary<string, string> secret
             {
                 var secretsResponse = new Autogenerated.SecretResponse();
                 secretsResponse.Secrets[secret.Key] = secret.Value;
-                getBulkSecretResponse.Data.Add(secret.Key, secretsResponse);
+                getBulkSecretResponse.Data.Add("IgnoredKey" + secret.Key, secretsResponse);
```
Comment (rynowak) [inline test/Dapr.Extensions.Configuration.Test/DaprSecretStoreConfigurationProviderTest.cs:440]: Why this change in particular?

I don't quite get the connection between this and your fix. Can you add a comment?

Comment (Fabian-Schmidt) [reply test/Dapr.Extensions.Configuration.Test/DaprSecretStoreConfigurationProviderTest.cs:440]: Bulk Secret API returns `Dictionary<string,Dictionary<string,string>>`. The outer Dictonary values are ignored by `DaprSecretStoreConfigurationProvider`. In the test setup the inner and outer values are identically. This is not the case in real world usage. Therefore the test did not fail to highlight an issue. This change is to ensure the tests are failing when reading the outer dictonary key, as I did.

Comment (rynowak) [reply test/Dapr.Extensions.Configuration.Test/DaprSecretStoreConfigurationProviderTest.cs:440]: Thanks, can you please add a comment?

Comment (Fabian-Schmidt) [reply test/Dapr.Extensions.Configuration.Test/DaprSecretStoreConfigurationProviderTest.cs:441]: Done

**Review by rynowak — APPROVED** [review]
